### PR TITLE
Removed duplicates from tutorials (second edition)

### PR DESCRIPTION
**Pull request by bebatut**
Hi,

This is the work done in #407 but rebased to avoid breaking the website, again... I will review it tomorrow again, but at least it builds successfully and there is no conflicts..

Bérénice

### CODE REVIEW

File: topics/introduction/metadata.yaml [hunk 1]
```diff
@@ -92,6 +139,16 @@ material:
...
+    contributors:
+    -
+        name: "Saskia Hiltemann"
+        github_username: "shiltemann"
+    -
+        name: "Dave Clements"
+        github_username: "tnabtaf"
+    -
+        name: "Anthony Bretaudeau"
+        github_username: "abretaud"
```
Comment (nsoranzo) [inline topics/introduction/metadata.yaml]: Missing: pajanne, nsoranzo.

File: topics/introduction/metadata.yaml [hunk 2]
```diff
@@ -124,6 +195,19 @@ material:
     objectives:
     time_estimation:
     key_points:
+    contributors:
+    -
+        name: "Nicola Soranzo"
+        github_username: "nsoranzo"
+    -
+        name: "Clemens Blank"
+        github_username: "blankclemens"
```
Comment (nsoranzo) [inline topics/introduction/metadata.yaml]: Remove nsoranzo and blankclemens.

File: topics/dev/metadata.yaml [hunk 3]
```diff
@@ -321,51 +461,3 @@ maintainers:
     name: "Bérénice Batut"
     github_username: "bebatut"
     email: ""
```
[the remark this answers is not in the document]
Comment (bebatut) [reply topics/dev/metadata.yaml]: I know but I was not sure for the topic. It is why I reopen the discussion about the maintainers #489...

File: _layouts/introduction_slides.html [hunk 4]
```diff
@@ -22,3 +21,7 @@
         {% endif %}
     {% endif %}
 {% endfor %}
+
+---
+
+## Thank you
```
[the remark this answers is not in the document]
Comment (bebatut) [reply _layouts/introduction_slides.html]: I will change that in a next PR

File: topics/dev/metadata.yaml [hunk 5]
```diff
@@ -284,13 +414,23 @@ material:
     - "Understand the various top-level files and modules in Galaxy."
     - "Understand how dependencies work in Galaxy's frontend and backend."
     time_estimation: "2.5h"
-    requirements: []
+    requirements:
```
[the remark this answers is not in the document]
Comment (bebatut) [reply topics/dev/metadata.yaml]: The linting of YAML will be fixed soon 😄

File: topics/introduction/metadata.yaml [hunk 6]
```diff
@@ -11,6 +11,16 @@ material:
...
+    contributors:
+    -
+        name: "Bérénice Batut"
+        github_username: "bebatut"
+    -
+        name: "Saskia Hiltemann"
+        github_username: "shiltemann"
+    -
+        name: "Anne Pajon"
+        github_username: "pajanne"
```
[the remark this answers is not in the document]
Comment (bebatut) [reply topics/introduction/metadata.yaml]: I am sorry...
It was not always easy to find who contributed to what

File: topics/introduction/metadata.yaml [hunk 7]
```diff
@@ -60,6 +89,13 @@ material:
     - "Workflows enable you to repeat your analysis on different data"
     - "Galaxy can connect to external sources for data import and visualization purposes"
     - "Galaxy provides ways to share your results and methods with others"
+    contributors:
+    -
+        name: "Anne Pajon"
+        github_username: "pajanne"
+    -
+        name: "Saskia Hiltemann"
+        github_username: "shiltemann"
```
[the remark this answers is not in the document]
Comment (bebatut) [reply topics/introduction/metadata.yaml]: Thanks

File: topics/introduction/metadata.yaml (hunk 1 above)
Comment (bebatut) [reply topics/introduction/metadata.yaml]: Thanks

File: topics/introduction/metadata.yaml (hunk 2 above)
Comment (bebatut) [reply topics/introduction/metadata.yaml]: Done

**Comment by bebatut** [comment]
Thanks a lot @nsoranzo for these comments

**Comment by nsoranzo** [comment]
Thanks @bebatut for the fixes! :+1:

**Pull request merged**